infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/pvc/PVCSubPathHelper.java

```diff
@@ -145,10 +146,11 @@ void execute(String workspaceId, String[] commandBase, String... arguments) {
       final Pod finished = deployments.wait(podName, WAIT_POD_TIMEOUT_MIN, POD_PREDICATE::apply);
       PodStatus finishedStatus = finished.getStatus();
       if (POD_PHASE_FAILED.equals(finishedStatus.getPhase())) {
+        String logs = deployments.getPodLogs(podName);
```

Comment:
Please consider adding the corresponding `verify` line in `PVCSubPathHelperTest`

Reply:
Done.